docs/pages/blog/masonry-post.md

```diff
@@ -0,0 +1,141 @@
+---
+title: The Journey of developing the MUI Masonry component
+description: Benny Joo recaps the development process of the MUI Masonry component.
+date: 2021-09-17T00:00:00.000Z
+authors: ['bennyjoo"]
+card: true
+---
+
+## Request from the community
+
+Just over two years ago, a [request](https://github.com/mui-org/material-ui/issues/17000) for a masonry component was opened. Upvoted by around 200 MUI users, this was one of the most popular component requests so far.
+
+![Screenshot 2021-09-15 at 09 13 33](https://user-images.githubusercontent.com/32841130/133396518-74c7792e-e4bc-403d-aad4-2585a7876206.png)
```

Comment:
I think that we should 1. host the image, 2. remove this random alt, 3. have the image in high-resolution (x2).